bin/run_unittests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+WORKDIR=${WORKDIR:=`pwd`}
+PYTHONPATH=$WORKDIR/
+export PYTHONPATH
```

Comment:
if root directory is the location of your source files, there is no sense in this PYTHONPATH export
Or, as I can see you have `src` directory so it probably must be the source path and look like this: PYTHONPATH=$WORKDIR/src